src/vstart.sh

```diff
@@ -232,7 +233,8 @@ options:
 	-g --gssapi enable Kerberos/GSSApi authentication
 	-G disable Kerberos/GSSApi authentication
 	--hitset <pool> <hit_set_type>: enable hitset tracking
-	-e : create an erasure pool
+    -e : create an erasure pool
```

Comment:
This line shouldn't be touched -- likely a stray indentation change here.  Note that these are indented by a tab character.

Reply:
Yeah it does follow tab indentation. Maybe the indentation just looks a bit weird over here in the file changes but it does look fine in the actual file.

Reply:
I checked out this PR and it looks the same locally:
```
	--hitset <pool> <hit_set_type>: enable hitset tracking
    -e : create an erasure pool
    --exporter: start the ceph-exporter daemon
	-o config add extra config parameters to all sections
```
The first and fourth lines are prefixed with a tab.  The second and third lines are prefixed with 4 spaces.